docs/reference/searchable-snapshots/index.asciidoc

```diff
@@ -293,6 +293,14 @@ repository. If you wish to search data across multiple regions, configure
 multiple clusters and use <<modules-cross-cluster-search,{ccs}>> or
 <<xpack-ccr,{ccr}>> instead of {search-snaps}.
 
+It's worth noting that if a searchable snapshot index has no replicas (as is the default
+in the cold tier), then when the node hosting it is shut down, allocation will immediately
+try to relocate the index to a new node in order to maximize availability. For fully mounted
+indices this will result in the new node downloading the entire index snapshot from
+the cloud repository, which might be expensive especially during rolling restarts. Temporarily
```

Comment:
I think we generally expect those costs to be very low and only exceptionally for it to have real cost implications, so I'd like to remove the cost mention:

```suggestion
the cloud repository. Temporarily
```

Reply:
👍🏻